### DISCUSSION TITLE
dKron cron schedule on debian machine

### DISCUSSION THREAD

**Discussion by ihershkovitz** [dkron-io/dkron#1291] [Q&A] [answered]
Hi ,
Installed dKron on my Debian instance.
I have create a job to run using the following cron : 0 57 14 1-7 * 0 ->
Meaning: "At 02:57 PM, between day 1 and 7 of the month, and on Sunday"
I assumed that since the 1st Sunday of the month (March) has passed, that dKron would set the next run to April 2nd.
Instead dKron set the next run to : 3/12/2023, 11:57:00 AM

Would appreciate any advice
Thanks

**Comment by yvanoers** [comment, marked as answer]
From the [Crontab manual](https://linux.die.net/man/5/crontab):
> Note: The day of a command's execution can be specified by two fields - day of month, and day of week. If both fields are restricted (ie, aren't *), the command will be run when either field matches the current time. For example,
"30 4 1,15 * 5" would cause a command to be run at 4:30 am on the 1st and 15th of each month, plus every Friday.

So it would be expected that it schedules on every Sunday.

That it schedules at the 11th hours is definitely wrong, but I'm hoping that's a typo on your part.

**Reply by ihershkovitz** [reply]
Thanks for the clarification
